sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 1.0.1 (2022-12-01)
```

Comment:
Actually I think we haven't release the stable version for Dns, could you please double check and update the changelog?